client/me/concierge/shared/primary-header.js

```diff
@@ -16,6 +16,7 @@ class PrimaryHeader extends Component {
 					displayAt="2023-12-26 00:00Z"
 					closesAt="2023-12-31 00:00Z"
 					reopensAt="2024-01-02 07:00Z"
+					isGM={ false }
```

Comment:
Explicitly adding the `isGM` prop here (instead of just omitting it) for easier discovery of the feature.